pkg/cmd/cli/cmd/status.go

```diff
@@ -59,11 +60,15 @@ type StatusOptions struct {
 // NewCmdStatus implements the OpenShift cli status command.
 func NewCmdStatus(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
 	opts := &StatusOptions{}
+	cmdName := fullName
+	if strings.HasSuffix(fullName, name) {
```

Comment:
then that's not the right value to use. We should not assume that stripping the last segment off the command will give us the root cli command name